src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -221,6 +221,7 @@ For more details on session replay, see the following sections:
     * In [Errors inbox](/docs/errors-inbox/errors-inbox/), filter your views so that you always see a replay. By default, browser event data is stored 8 days, but actual data retention depends on your account.
     * If you're using the copy/paste deployment method, make sure you [updated your agent code](#update-snippet).
     * If you’re using the APM deployment method, you may need to wait a while for session replay to be automatically enabled. If you’re still not seeing data, try restarting the agent.
+    * [Dropping](https://docs.newrelic.com/docs/data-apis/manage-data/drop-data-using-nerdgraph/) or not reporting PageView event data can prevent Session Replays from displaying in the UI.
```

Comment:
Suggestion: 
Ensure that you are [dropping](https://docs.newrelic.com/docs/data-apis/manage-data/drop-data-using-nerdgraph/) the PageView event data or not reporting it to prevent Session Replays from displaying in the UI.

Reply:
Since this is a troubleshooting section focusing on getting Session Replays to appear in the UI, I feel like the suggestion comes across as if we're recommending users to drop PageView data.
In keeping consistent format with other bullet points in the section, how about:
"If you're [dropping](https://docs.newrelic.com/docs/data-apis/manage-data/drop-data-using-nerdgraph/) PageView event data, Session Replays may not display in the UI."